code/src/pages/MovieList.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+
+export const MovieList = () => {
+  const [movies, setMovies] = useState([])
+
+  useEffect(() => {
+
+    fetch('https://api.themoviedb.org/3/movie/popular?api_key=db37f43a686cbd6ad83202f3d97ecfb5&language=en-US&page=1')
+      .then((res) => res.json())
+      .then((json) => {
+        setMovies(json.results)
+      })
+  }, [])
+
+  return (
+    <div className="outerDiv">
+      {movies.map((results) => (
+
+        <div key={results.id} className="movieDisplay">
```

Comment:
you dont really need the div around here. put the key and className directly on Link